vaadin-notification-flow-parent/vaadin-notification-testbench/src/main/java/com/vaadin/flow/component/notification/testbench/NotificationElement.java

```diff
@@ -60,6 +62,19 @@ public SearchContext getContext() {
      * @return the card for the notification
      */
     private TestBenchElement getCard() {
-        return getPropertyElement("_card");
+        var card = getPropertyElement("_card");
+
+        try {
+            // Wait until the content becomes visible
+            waitUntil(driver -> {
+                var hasVisbleText = !card.getText().isEmpty();
+                var hasVisibleChildren = card.findElements(By.cssSelector("*"))
+                        .stream().anyMatch(element -> element.isDisplayed());
+                return hasVisbleText || hasVisibleChildren;
+            }, 100);
```

Comment:
The timeout would only occur after 100 seconds which seems excessive for this case (waiting for animation to finish). Did you intend to use 100ms?

Reply:
Good catch. Changed to 1 second